src/app/fsat/fsat-fluid/fsat-fluid.component.html

```diff
@@ -31,6 +31,21 @@ <h3>Fluid</h3>
       </span>
     </div>
 
+    <div class="form-group"
+      [ngClass]="{'indicate-different': isSpecificHeatRatioDifferent(), 'invalid':gasDensityForm.controls.specificHeatRatio.invalid}">
+      <label for="{{'specificHeatRatio_'+idString}}">Specific Heat Ratio (&gamma;)</label>
+      <input name="specificHeatRatio" type="number" min="0" step="any" class="form-control"
+        formControlName="specificHeatRatio" id="{{'specificHeatRatio_'+idString}}" (input)="save()"
+        (focus)="focusField('specificHeatRatio')" onfocus="this.select();" [readonly]="!selected">
+      <span class="alert-danger pull-right small"
+        *ngIf="gasDensityForm.controls.specificHeatRatio.invalid && !gasDensityForm.controls.specificHeatRatio.pristine">
+        <span *ngIf="gasDensityForm.controls.specificHeatRatio.errors.required">Value Required</span>
+        <span *ngIf="gasDensityForm.controls.specificHeatRatio.errors.max">Value can't be less than
```

Comment:
"Value can't be less than max" think it should say "Value can't be greater than"